src/components/Tooltips/TooltipWithSubtitle.tsx

```diff
@@ -0,0 +1,35 @@
+import { Tooltip, TooltipProps } from "antd";
+import React, { ReactElement, ReactNode, useRef } from "react";
+
+/**
+ * A wrapper around the Ant Design Tooltip that adds support for a subtitle (or
+ * list of subtitles) beneath the main tooltip text.
+ */
+export function TooltipWithSubtitle(
+  props: TooltipProps & { title: ReactNode; subtitle?: ReactNode; subtitleList?: ReactNode[] }
+): ReactElement {
+  const divRef = useRef<HTMLDivElement>(null);
+  return (
+    <Tooltip
+      {...props}
+      trigger={["hover", "focus"]}
+      title={
+        <>
+          <p style={{ margin: 0 }}>{props.title}</p>
+          {props.subtitle && <p style={{ margin: 0, fontSize: "12px" }}>{props.subtitle}</p>}
+          {props.subtitleList &&
+            props.subtitleList.map((text, i) => (
+              <p key={i} style={{ margin: 0, fontSize: "12px" }}>
+                {text}
+              </p>
+            ))}
+        </>
+      }
+      getPopupContainer={() => divRef.current ?? document.body}
+    >
+      {props.children}
+      {/* TODO: include invisible text here copying the tooltip? */}
+      <div ref={divRef}></div>
```

Comment:
I did make one change from the original here where the `divRef` used to place the Tooltip in the document hierarchy is inside of the Tooltip component. This fixes a bug where radio buttons wrapped in this component would have incorrect rounding.